ruby/red-arrow/test/test-record-batch-file-reader.rb

```diff
@@ -112,4 +112,18 @@ class RecordBatchFileReaderTest < Test::Unit::TestCase
       end
     end
   end
+
+  sub_test_case("#each") do
+    test("without block") do
+      buffer = Arrow::ResizableBuffer.new(1024)
+      Arrow::Table.new(number: [1, 2, 3]).save(buffer)
+      Arrow::BufferInputStream.open(buffer) do |input|
+        reader = Arrow::RecordBatchFileReader.new(input)
+        assert_equal([
+                       Arrow::RecordBatch.new(number: [1, 2, 3]),
+                     ],
+                     reader.each.to_a)
+      end
+    end
```

Comment:
```suggestion
    test("without block") do
      buffer = Arrow::ResizableBuffer.new(1024)
      Arrow::Table.new(number: [1, 2, 3]).save(buffer)
      Arrow::BufferInputStream.open(buffer) do |input|
        reader = Arrow::RecordBatchFileReader.new(input)
        assert_equal([
                       Arrow::RecordBatch.new(number: [1, 2, 3]),
                     ],
                     reader.each.to_a)
      end
    end

    test("without block, test size") do
      buffer = Arrow::ResizableBuffer.new(1024)
      Arrow::Table.new(number: [1, 2, 3]).save(buffer)
      Arrow::BufferInputStream.open(buffer) do |input|
        reader = Arrow::RecordBatchFileReader.new(input)
        assert_equal 1, reader.each.size
      end
    end
```

Added test "without block, test size".

Reply:
I'll test them at once for easy to debug.